package.json

```diff
@@ -36,6 +36,7 @@
     "@patternfly/react-styles": "4.92.6",
     "@patternfly/react-table": "4.113.0",
     "@rhoas/app-services-ui-components": "2.20.9",
+    "@patternfly/react-topology": "^4.91.27",
```

Comment:
Could you switch to the latest version of this package? Also with an exact number, no ranges.

Reply:
Nitpicking, I would also move it up to keep the alphabetical order of packages.